pkg/operator/staticpod/controllers.go

```diff
@@ -75,6 +75,9 @@ type staticPodOperatorControllerBuilder struct {
 	readyzEndpoint                string
 	pdbUnhealthyPodEvictionPolicy *v1.UnhealthyPodEvictionPolicyType
 	guardCreateConditionalFunc    func() (bool, bool, error)
+
+	// should revision install information
```

Comment:
not sure what this comment is supposed to mean

Reply:
Here I was trying to provide a description for the newly added field `shouldRevisionInstall`. May be I should be using a more meaningful description.